GladiusEx.lua

```diff
@@ -816,11 +822,14 @@ function GladiusEx:CheckUnitSpecialization(unit)
 end
 
 function GladiusEx:UpdateUnitSpecialization(unit, specID)
+	log("updateunit:"..unit)
 	local _, class, spec

```

Comment:
I can probably remove `class` from here/cleanup the `GetSpecializationInfoByID` below.